helpers/basename.js

```diff
@@ -1,5 +1,3 @@
-const {basename} = require('path');
-
 module.exports = function (path) {
-	return basename(path).split('.')[0];
+  return path.split(/[\/\.]/g)[0];
```

Comment:
This still breaks on Windows for me (which is my main development environment). Could we potentially change this back to the `const {basename} = require('path')` or:

```suggestion
  return path.split(/[\/\.\\]/g)[0];
```